content/en/docs/Tasks/creating-a-catalog.md

```diff
@@ -17,15 +18,62 @@ be made available to install in a cluster. You can make your operator bundle ava
 it to a catalog, packaging the catalog in a container image, and then using that image reference in the `CatalogSource`.
 This image contains all of the metadata required for OLM to manage the lifecycle of all of the operators it contains.
 
-OLM uses a plaintext [file-based catalog][file-based-catalog-spec] format (JSON or YAML) to store these records in a Catalog, and `opm` has tooling
-that helps initialize a Catalog, add new operators into it, and then validate that the catalog is valid. Let's walk
-through a simple example.
+OLM uses a plaintext [file-based catalog][file-based-catalog-spec] format (JSON or YAML) to store these records in a Catalog, and there are two approaches we can take to creating a Catalog, adding operators to it, and validating it. 
+Let's walk through a simple example for both approaches.
 
+### Catalog Creation Using Veneers
+[Veneers][veneers-doc] are a purpose-built simplification of [File-Based Catalogs][file-based-catalog-spec] to ease common catalog operations.  For this example, we'll be using the [semver veneer][semver-veneer-doc].
+>Note: We strongly recommend that authors create and maintain their veneers in a version-controlled environment discrete from their generated catalogs.  Further, we recommend that authors focus on the veneer as the sole artifact connecting the operator to the catalog (even going so far as only generating the file-based catalog during CI/CD tooling so it is only provided for catalog contribution.)
+
+#### Catalog Creation
+First we need to create the Catalog hierarchy and Dockerfile for generating the image
+```sh
+$ mkdir -p cool-catalog/example-operator
+$ opm generate dockerfile cool-catalog
```

Comment:
Consider splitting up these command into separate code blocks for easier readability. Also, the Kubernetes style guide says not to use the `$` for the command prompt.